ultraLatency.py

```diff
@@ -182,6 +190,7 @@
         pass
     configureScanModule(ohboard, options.gtx, mode, mask,
                         scanmin=LATENCY_MIN, scanmax=LATENCY_MAX,
+                        stepsize=step,
```

Comment:
Yeah, sure, I didn't mean for all the commits to send PR notifications every time. Just needed to update my repo to download it in a different location. 

I implemented some changes run_scans.py but couldn't test them yet (my $DATA_PATH didn't have permissions yet). Soon ...